xml/art_slert_hw_testing.xml

```diff
@@ -176,95 +176,95 @@
     <title>Determining Latencies with the Hardware Latency Detector</title>
 
     <para> The <emphasis>hardware latency detector</emphasis>
-        (<filename>hwlat_detector.ko</filename>) is installed with the
-      real-time kernel (the <package>kernel-rt</package> package).
+        (<command>hwlatdetect</command>) is a program that controls the kernel
+      latency detector module (hwlat_detector.ko), installed with the
+      real-time debug kernel (the <package>kernel-rt_debug</package> package).
       It is a special kernel module that is used to detect large system
       latencies. It is caused by the behavior of underlying hardware or
       firmware, independent of Linux itself (system management
       interrupts, SMIs). </para>
 
     <para> The hardware latency detector module works by making all CPUs
       busy for a configurable amounts of time. This is done by calling
-      <systemitem class="other" otherclass="function"
-       >stop_machine()</systemitem>, polling the CPU Time Stamp Counter (TSC)
+      <systemitem class="other" otherclass="function">stop_machine()</systemitem>, polling the CPU Time Stamp Counter (TSC)
       for some period, and then looking for gaps in the TSC data.
       These gaps indicate times when the polling was disturbed by
       System Management Interrupts (SMI).</para>
```

Comment:
```suggestion
      SMIs.</para>
```

Reply:
Since we just defined this abbreviation in the previous paragraph, I think we do not need to immediately give the full version again.